app/controllers/providers/proceeding_merits_task/specific_issue_controller.rb

```diff
@@ -0,0 +1,34 @@
+module Providers
+  module ProceedingMeritsTask
+    class SpecificIssueController < ProviderBaseController
+      def show
+        @form = SpecificIssueForm.new(model: specific_issue)
+      end
+
+      def update
+        @form = SpecificIssueForm.new(form_params.merge(proceeding_id: proceeding.id))
+        render :show unless update_task_save_continue_or_draft(proceeding.ccms_code.to_sym, :specific_issue)
+      end
+
+    private
+
+      def specific_issue
+        @specific_issue ||= proceeding.specific_issue || proceeding.build_specific_issue
+      end
+
+      def proceeding
+        @proceeding ||= Proceeding.find(params[:merits_task_list_id])
+      end
+
+      def legal_aid_application
+        @legal_aid_application ||= proceeding.legal_aid_application
+      end
```

Comment:
How do you feel about us scoping proceedings by legal aid application (that's assigned in the `Providers::ApplicationDependable` module included in `ProvidersBaseController`) instead?

```suggestion
      def proceeding
        @proceeding ||= legal_aid_application.proceedings.find(params[:merits_task_list_id]
      end
```

Reply:
This one won't work without changes... a while ago we had conversations about URLs getting 'too long' and could we 'shorten' them 🙄 
Because tasks on the task list were needed by proceeding they would have ended up as `/providers/applications/436c3998-606f-456b-84be-d9c3b742548e/merits_task_list/1e516e08-a991-48af-b3da-63ffa7655a6c/specific_issue`
So we truncated the url (when in the task list) to be `/providers/merits_task_list/1e516e08-a991-48af-b3da-63ffa7655a6c/chances_of_success`
Which means that partial doesn't have the legal_aid_application_id available 😅 
Hence (I assume) why we load the proceeding and then extrapolate the application from it :+1:

Reply:
URLs being too long is a new one!

Thanks for the context, I eagerly await the days of even shorter URLs 😆 

https://bit.ly/3Dj7lmU